### DISCUSSION TITLE
Chocolatey version is not matching? Is it normal?

### DISCUSSION THREAD

**Discussion by ekozcifci** [Devolutions/UniGetUI#3128] [Q&A]
As you see, versions are not same with chocolatey, is it a bug? i am newbie so im just confused.

![image](https://github.com/user-attachments/assets/4d1ce5e4-d65f-4ea9-8df1-8a5d9e97de15)

**Comment by marticliment** [comment]
Chocolatey may sometimes take a while to update its package indexes, is the update recent?

**Reply by ekozcifci** [reply]
No, it is not. It is updated on 12 December. Also same bug happens on almost every package i searched on. Not package spesific issue. 
FYI, i just miss-created this topic, i created same on on issue section too, sorry.

**Reply by marticliment** [reply]
I will investigate

**Comment by albino1** [comment]
I've just installed UniGetUI for the first time, and I'm also seeing this problem.  Sometimes the chocolatey version shown in the GUI is an old version of the program, and not the current one.  Here's an example using the program SABnzbd, which should be defaulting to 4.4.1 but for some reason UniGetUI shows 4.3.3 by default:

<details>
  <summary>Searching for "sabnzbd" in Discover Packages</summary>

![wingetui1](https://github.com/user-attachments/assets/8e62e553-8ede-424d-96ca-2e93d169b9b4)

</details>

<details>
  <summary>Verifying that it's set to Latest and that 4.4.1 is available in the options</summary>

![wingetui2](https://github.com/user-attachments/assets/88f435bd-7cb7-4834-a254-5432c2214c0a)

</details>

Here's me checking on the command line with chocolatey and you can see that it automatically selects 4.4.1 and not 4.3.3 as the latest stable release:

```
> choco install sabnzbd
Chocolatey v2.4.2
Installing the following packages:
sabnzbd
By installing, you accept licenses for the packages.
Downloading package from source 'https://chocolatey.org/api/v2/'
Progress: Downloading SABnzbd 4.4.1... 100%

sabnzbd v4.4.1 [Approved]
```

Chocolatey URL for `sabnzbd` when searching the chocolatey website:

https://community.chocolatey.org/packages?q=sabnzbd

https://community.chocolatey.org/packages/SABnzbd/4.4.1

Checking the [Version History](https://community.chocolatey.org/packages/SABnzbd/4.4.1#versionhistory) we can see that 4.4.1 was added on December 20th so it shouldn't be affected by the package indexing.

This is using UniGetUI 3.1.6 installed from Github and not from the Microsoft Store.  In case it's relevant, I already had chocolatey installed before installing UniGetUI.

**Reply by marticliment** [reply]
This issue will be fixed in next version

**Reply by albino1** [reply]
@marticliment I saw in the release notes what looked like a fix, but I just checked and it's still showing the wrong version for my previous example:

![uniget](https://github.com/user-attachments/assets/8b0b0c3a-d6b8-471a-a330-92dbd8b9d110)

Just a heads up in case you thought it was solved.

**Reply by marticliment** [reply]
I have fixed something, because I was not able to reproduce the issue anymore. Nevertheless, I will look into the issue again
